hopsandcp/dcpslave.cpp

```diff
@@ -0,0 +1,244 @@
+#include "dcpslave.h"
+#include "utilities.h"
+
+#include <dcp/helper/Helper.hpp>
```

Comment:
If dcp is bundled inside Hopsan, then maybe use #include "path/here"  instead of < >, not a big deal